### DISCUSSION TITLE
Component Z-axis too high!

### DISCUSSION THREAD

**Discussion by Baleandor** [ant-design/ant-design#44214] [Q&A] [answered]
EDIT: added repo link
Howdy!
 I am using the checkbox and star rating components and have a sticky header. The 2 components are visible when the header is over them. How can I prevent that? I tried changing their z-axis values but it didn't help.

Repo link: https://github.com/Baleandor/product-listings-page

Thanks!

**Comment by kiner-tang** [comment]
Hello @Baleandor . Please provide a online reproduction by forking codesandbox of [antd@5.x](https://u.ant.design/codesandbox-repro) or [antd@4.x](https://u.ant.design/codesandbox-repro-4x), or provide a minimal GitHub repository. Issues labeled by `Need Reproduce` will be closed if no activities in 3 days.

你好 @Baleandor, 我们需要你提供一个在线的重现实例以便于我们帮你排查问题。你可以通过点击这里创建一个 [antd@5.x](https://u.ant.design/codesandbox-repro) 或 [antd@4.x](https://u.ant.design/codesandbox-repro-4x) 的 codesandbox，或者提供一个最小化的 GitHub 仓库。3 天内未跟进此 issue 将会被自动关闭。

![](https://gw.alipayobjects.com/zos/antfincdn/y9kwg7DVCd/reproduce.gif)

**Reply by Baleandor** [reply]
Post edited, repo link added. I apologize for forgetting.

**Reply by kiner-tang** [reply, marked as answer]
Maybe you can set 'z-10' on the header to solve this problem.
<img width="1491" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/1406eac8-766f-48cd-a269-d86b790efa7d">

**Reply by Baleandor** [reply]
That works, thanks! I had the Navbar component itself have a z-10 but that didn't help.
